node/src/chain_spec.rs

```diff
@@ -175,19 +177,19 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
                 // SnowBridged Assets
                 vec![
                     (
-                        "Mangata".as_bytes().to_vec(),
-                        "MNG".as_bytes().to_vec(),
-                        "Mangata Asset".as_bytes().to_vec(),
+                        "Mangata".to_string().into_bytes(),
+                        "MNG".to_string().into_bytes(),
+                        "Mangata Asset".to_string().into_bytes(),
                         18u32,
                         0u32,
                         H160::from_slice(&hex!["F8F7758FbcEfd546eAEff7dE24AFf666B6228e73"][..]),
-                        100_000_000__000_000_000_000_000_000u128,
+                        100_000_000_000_000_000_000_000_000u128,
```

Comment:
Can we suppress a warning here? The original constant `100_000_000__000_000_000_000_000_000u128` visually separates integer and decimal parts of the total amount